nvflare/app_common/widgets/validation_json_generator.py

```diff
@@ -58,7 +72,6 @@ def handle_event(self, event_type: str, fl_ctx: FLContext):
             if val_results:
                 try:
                     dxo = from_shareable(val_results)
-                    dxo.validate()
```

Comment:
why don't we need to validate the dxo here anymore?

Reply:
good question, we already did that in creation: https://github.com/NVIDIA/NVFlare/blob/main/nvflare/apis/dxo.py#L65